frontend/src/router.js

```diff
@@ -69,7 +71,34 @@ const router = new VueRouter({
       path: '/token-expired',
       name: 'backend-session-expired',
       component: BackendSessionExpired
-    }
+    },
+    {
+      path: '/',
+      component: RouterView,
+      children: [
+        {
+          path: 'inbox',
+          name: 'inbox',
```

Comment:
nitpik - should we be consistent in the naming for the secure messaging inbox. On the student admin side it's /exchange. Was thinking it would be good to be consistent. Though I'm fine with inbox too, we can rename it later if needed.